server/seed/demoSeed/generateComments.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
+
+const {
+  Types: { ObjectId }
+} = mongoose;
+
+const generateComments = listData => {
+  const comments = [];
+  for (let i = 0; i < listData.length; i += 1) {
```

Comment:
store this lenght in a constanst before loop
I've read that is better fore loop perfomance. Cause it's cached above the loop and it doesn't have to be evaluated in every loop iteration.